addons/website_payment/static/src/interactions/website_payment_donation.js

```diff
@@ -0,0 +1,33 @@
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+
+export class WebsitePaymentDonation extends Interaction {
+    static selector = ".o_donation_payment_form";
+    dynamicContent = {
+        ".o_amount_input": { "t-on-focus": this.onFocus },
+        "#donation_comment_checkbox": { "t-on-change": this.onChange },
+    }
+
+    /**
+     * @param {Event} ev
+     */
+    onChange(ev) {
```

Comment:
I don't think changing the methods with generic names is a good idea. Even in a small interaction like this, having a clear name helps read the code. (Those generic names tend to imply the action is done on `this.el`, which is not the case.)

Reply:
you didn't change the name here :eyes: